### ISSUE TITLE
Import fails with scvi-tools >= 0.20

### ISSUE THREAD

**Issue by VladimirShitov** [YosefLab/mrvi_archive#8]
First of all, thank you for a great tool! Unfortunately, it is impossible to run import it with the newest version of scvi-tool. When I run
```python
import mrvi
```
I get the following error:
```python
...
----> 5 from scvi.module.base import BaseModuleClass, LossRecorder, auto_move_data
      6 from scvi.nn import one_hot
      7 from torch.distributions import kl_divergence as kl

ImportError: cannot import name 'LossRecorder' from 'scvi.module.base' 
```

The reason is this line in your code:
 https://github.com/YosefLab/mrvi/blob/d1934e4889bbf383e411d2d39558488e1568fb0c/mrvi/_module.py#L5

In scvi-tools v 0.20 `LossRecorder` was renamed to `LossOutput`. I suggest checking the version of scvi-tool in the code before importing loss recorder or simply fixing it to <0.20 in requirements

Have a great day!

**Comment by VladimirShitov** [comment]
Upd, I noticed that there is a PR solving this issue: https://github.com/YosefLab/mrvi/pull/6

**Comment by SamuelCWJ** [comment]
> First of all, thank you for a great tool! Unfortunately, it is impossible to run import it with the newest version of scvi-tool. When I run
> 
> ```python
> import mrvi
> ```
> 
> I get the following error:
> 
> ```python
> ...
> ----> 5 from scvi.module.base import BaseModuleClass, LossRecorder, auto_move_data
>       6 from scvi.nn import one_hot
>       7 from torch.distributions import kl_divergence as kl
> 
> ImportError: cannot import name 'LossRecorder' from 'scvi.module.base' 
> ```
> 
> The reason is this line in your code:
> 
> https://github.com/YosefLab/mrvi/blob/d1934e4889bbf383e411d2d39558488e1568fb0c/mrvi/_module.py#L5
> 
> In scvi-tools v 0.20 `LossRecorder` was renamed to `LossOutput`. I suggest checking the version of scvi-tool in the code before importing loss recorder or simply fixing it to <0.20 in requirements
> 
> Have a great day!

Hi!

I just encountered this same error trying to import MrVI. I'm not really familiar with using python.. Please could you help me, how may I work around this?

Thank you so much!

Samuel

**Comment by martinkim0** [comment]
Hi, this is being addressed in #11 and will be re-releasing after this merges. Thanks for your patience.
